### ISSUE TITLE
How to use angularjs date filter with angular-translate

### ISSUE THREAD

**Issue by luizfilipe** [angular-translate/angular-translate#860]
It is possible to use post compiling to process date filter with translate directive, like follows?

``` json
"TODAY": "Today is: {{ date | date:'MM/dd/yyyy' }}"
```

@ HTML

``` html
<p translate="TODAY" translate-values="{date: {{new Date()}}}">
```

**Comment by LiamK** [comment]
Hi, @luizfilipe, 
What was the solution?

**Comment by bborowin** [comment]
@luizfilipe could you summarize your solution pls?

**Comment by luizfilipe** [comment]
@LiamK and @bborowin w8 for a few hours I'll get the changed code asap and explain my solution after..

**Comment by jasonvlambert** [comment]
It didn't work for me either using the translate-values={myDateVar: someScopeObject.myDateVar} format.

I eventually found that it did work if I used the translate-values={{translationData}} format, where         $scope.translationData = {
            myDateVar: someDateValue
};


translate="CLOSING_ON" translate-values="{{translationData}}"

**Comment by envision** [comment]
Still no solution?

**Comment by dani-inside** [comment]
I was facing the same issue and found a solution for using the date filter in translate-values directive. It's about using brackets to make the statements correct. 
`<div translate="translateKey" translate-values="{date: (model.date | date:'dd.MM.yyyy')}"></div>`
